docs/sphinx/methods/dsrg.rst

```diff
@@ -679,7 +679,43 @@ We have implemented the following choices for the zeroth-order Hamiltonian.
   cvcv, cvvc, vcvc, vccv, avav, avva, vava, and vaav.
   The computation procedure is similar to that of Dyall Hamiltonian.
 
-6. Examples
+6. Restart iterative MRDSRG from a previous computation
++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The convergence of iterative MRDSRG [e.g., MR-LDSRG(2)] can be greatly improved if it starts from good initial guesses
+(e.g., from loosely converged amplitudes or those of a near-by geometry).
+The amplitudes can be dumped to the current working directory on disk for later use by turning on the ``DSRG_DUMP_AMPS`` keyword.
+These amplitudes are stored in a binary file using Ambit (version later than 06/30/2020).
+For example, T1 amplitudes are stored as ``forte.mrdsrg.spin.t1.bin`` for the spin-integrated code
+and ``forte.mrdsrg.adapted.t1.bin`` for spin-adapted code (i.e., `correlation_solver` set to `sa-mrdsrg`).
+To read amplitudes in the current directory (must follow the same file name convention),
+the user needs to invoke the ``DSRG_READ_AMPS`` keyword.
+
+.. note::
+  In general, we should make sure the orbital phases are consistent between reading and writing amplitudes.
+  For example, the following shows part of the input to ensure the coefficient of the first AO being positive for all MOs. ::
+
+    ...
+    Escf, wfn = energy('scf', return_wfn=True)
```

Comment:
This procedure should be implemented in forte. (we might have to coordinate with Nan's changes to the QC repo).

Reply:
Ok, I found some C++ code below. So maybe things just need to be harmonized.